WordPress/src/main/res/layout/activity_log_list_progress_item.xml

```diff
@@ -1,38 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              xmlns:app="http://schemas.android.com/apk/res-auto"
               xmlns:tools="http://schemas.android.com/tools"
+              android:id="@+id/activity_content_container"
               android:layout_width="match_parent"
               android:layout_height="wrap_content"
-              xmlns:app="http://schemas.android.com/apk/res-auto"
+              android:background="?android:selectableItemBackground"
```

Comment:
This layout has the same touch feedback issue as `activity_log_list_event_item.xml`.  We can change `android:background` to `android:foreground` in the parent `LinearLayout` for a quick fix.  This layout is *very* similar to `activity_log_list_event_item.xml` actually.  We should try to combine them into a single file.

Reply:
I've fixed the background issue. I'd rather not combine the two layouts for the same reasons I wouldn't combine the view holders. I'd rather have some code duplication in favour of readability. I think it's better when the future dev doesn't have to look for the logic in the layout in order to see what's actually showing to the user (depends on the scale of course). We could use styles to cover some of the duplication.